examples/create-react-app/src/components/period-selector-dialog.js

```diff
@@ -8,6 +8,12 @@ import PeriodSelectorDialog from '@dhis2/d2-ui-period-selector-dialog';
 
 class PeriodSelectorDialogExample extends Component {
     state = {
+        // example of initially selected periods
+        periods: [
+            { id: 'TODAY', name: 'Today' },
+            { id: 'YESTERDAY', name: 'Yesterday' },
+            { id: 'LAST_3_DAYS', name: 'Last 3 days' },
```

Comment:
Does the name (which seems to be the translated string used in a label) need to be passed via props?
I expect typically the app to only pass the list of ids